beacon_node/network/src/metrics.rs

```diff
@@ -374,11 +375,31 @@ pub static PEERS_PER_SYNC_TYPE: LazyLock<Result<IntGaugeVec>> = LazyLock::new(||
 });
 pub static PEERS_PER_COLUMN_SUBNET: LazyLock<Result<IntGaugeVec>> = LazyLock::new(|| {
     try_create_int_gauge_vec(
-        "peers_per_column_subnet",
+        "sync_peers_per_column_subnet",
         "Number of connected peers per column subnet",
         &["subnet_id"],
     )
 });
+pub static PEERS_PER_CUSTODY_COLUMN_SUBNET: LazyLock<Result<IntGaugeVec>> = LazyLock::new(|| {
+    try_create_int_gauge_vec(
+        "sync_peers_per_custody_column_subnet",
+        "Number of connected peers per custody column subnet",
+        &["subnet_id"],
+    )
+});
+pub static COLUMN_SUBNETS_WITH_ZERO_PEERS: LazyLock<Result<IntGauge>> = LazyLock::new(|| {
+    try_create_int_gauge(
+        "sync_column_subnets_with_zero_peers",
+        "Current count of total column subnets with zero peers",
+    )
+});
```

Comment:
Can't we just use `sync_peers_per_column_subnet` for this?

and the query:
```
count(sync_peers_per_column_subnet == 0)
```

Reply:
Yeah I mentioned that in the description, we can skip the metric if we are ok doing a Grafana query